lib/fizzy/types.hpp

```diff
@@ -51,6 +51,9 @@ using TableIdx = uint32_t;
 // https://webassembly.github.io/spec/core/syntax/modules.html#syntax-memidx
```

Comment:
BTW these links now go to 1.1 spec

Reply:
Do you want to open an issue/PR for us to direct-link to 1.0?

Reply:
https://github.com/wasmx/fizzy/issues/366